### PR TITLE
reformatted the login and register to not use formik and created Auth…

### PR DESCRIPTION
**Pull request by jess-lam**
…entication folder

# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by onepeso** [comment]
[Refactor Login/Signup Forms](https://trello.com/c/AFvY4BaW/29-refactor-login-signup-forms)

File: src/components/Authentication/ProtectedRoute.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            if (localStorage.getItem('token')){
+                return <Component/>;
+            } else {
+                return <Redirect to='/' />
+            }
+            }}
+        />
+    );
+};
+
```
Comment (onepeso) [inline src/components/Authentication/ProtectedRoute.js:20]: Looks great!

**Comment by onepeso** [comment]
@jess-lam If you haven't changed anything yet can we merge this, so the team can pull this to continue work? Also, don't worry just create another branch when you are working on making login and sign up hook up to backend.

**Review by onepeso — APPROVED** [review]
Great! I like the simple reformat of the login and register forms look DRY now :)

**Comment by jess-lam** [comment]
> @jess-lam If you haven't changed anything yet can we merge this, so the team can pull this to continue work? Also, don't worry just create another branch when you are working on making login and sign up hook up to backend.

**Review by TheNativeDev — APPROVED** [review]
This looks great

**Review by TheNativeDev — COMMENTED** [review]
Thank you so much for doing this for us!

**Pull request merged**